en/includes/guides/customize/actions/understanding-actions.md

```diff
@@ -0,0 +1,184 @@
+</ul>
+</td>
+</tr>
+<tr class="even">
+<td>Token management</td>
+<td>Pre issue access token</td>
+<td>Early access (beta)</td>
+<td>
+<ul>
+<li><p>Prevent tokens from being issued based on policy or change permissions, scopes.</p></li>
+<li><p>Add custom claims to access tokens.</p></li>
+<li><p>Change the access token and refresh token validity based on a rule.</p></li>
+</ul>
+</td>
+</tr>
+<tr class="odd">
+<td>Registration</td>
+<td>Pre registration</td>
+<td>September 2024</td>
+<td>
+<ul>
+<li><p>Deny registration by location.</p></li>
+<li><p>Set additional data in the user profile.</p></li>
+<li><p>Initiate an external service to trigger a verification such as an approval, biometrics, allowed lists, etc.</p></li>
+<li><p>Verify for sanctioned countries, initiate screening processes.</p></li>
+</ul>
+</td>
+</tr>
+<tr class="even">
+<td>Password update</td>
+<td>Pre password update</td>
+<td>September 2024</td>
+<td>
+<ul>
+<li><p>Challenge with an additional verification factor before allowing to update password.</p></li>
+<li><p>Integrate with credential intelligence services to verify the security of password, or check against allowed lists.</p></li>
+</ul>
+</td>
+</tr>
+<tr class="odd">
+<td>Profile update</td>
+<td>Pre profile update</td>
+<td>September 2024</td>
+<td>
+<ul>
+<li><p>Challenge with an additional verification factor before allowing to update sensitive attributes.</p></li>
+<li><p>Redirect to a third party service for verification.</p></li>
+<li><p>Verify for sanctioned countries, initiate screening processes.</p></li>
+</ul>
+</td>
+</tr>
+</tbody>
+</table>
+
+## How actions work
+
+Each type of action is executed at a specific extension point within a particular runtime flow of {{product_name}}. When an action is triggered, {{product_name}} calls your external service and waits for a response. Upon receiving the response, {{product_name}} performs any specified operations and continues with the flow.
+
+The following diagram illustrates the sequence of these steps:
+
+![how-actions-work]({{base_path}}/assets/img/guides/actions/how-actions-work.png){: width="650" style="display: block; margin: 0; border: 0px;"}
+
+### Request from {{product_name}} 
+
+{{product_name}} sends an HTTPS POST request to your external web service with a JSON payload that includes data relevant to the flow that triggered the action. The request consists of the following components:
+
+<table>
+<thead>
+<tr class="header">
+<th>Property</th>
+<th>Description</th>
+</tr>
+</thead>
+<tbody>
+<tr class="odd">
+<td>flowId</td>
+<td>
+<p>A unique correlation identifier for the flow (e.g., a login process involving input collection, authentication, and two-factor authentication) initiated and executed by {{product_name}}.</p>
+    <p>
+    !!! note
+        Currently, not all actions incorporate a flowId. The presence of a flowId depends on the type of action and the executed flow.
+    </p>
```

Comment:
@himash-wso2 this note is not well formatted. Please suggest the right html tags to follow.